flixkit.go

```diff
@@ -122,36 +91,29 @@ func (s *flixServiceImpl) GetFlixByID(ctx context.Context, templateID string) (*
 	return parsedTemplate, nil
 }
 
-func (t *FlowInteractionTemplate) IsScript() bool {
-	return t.Data.Type == "script"
-}
-
-func (t *FlowInteractionTemplate) IsTransaction() bool {
-	return t.Data.Type == "transaction"
-}
-
-func (t *FlowInteractionTemplate) GetAndReplaceCadenceImports(networkName string) (string, error) {
-	cadence := t.Data.Cadence
 
-	for dependencyAddress, contracts := range t.Data.Dependencies {
-		for contractName, networks := range contracts {
-			network, ok := networks[networkName]
-			if !ok {
-				return "", fmt.Errorf("network %s not found for contract %s", networkName, contractName)
-			}
+func (s *flixServiceImpl) GenFlixBinding(ctx context.Context, templateLocation string, lang string, isLocal bool) (string, error) {
+	var template string
+	var err error
+	if isLocal {
+		template, err = FetchFlixWithContextFromFile(s.config.FileReader, ctx, templateLocation)
```

Comment:
Don't think you need this anymore right? Aren't we grabbing it from the CLI with a file reader and passing it over.

Reply:
"grabbing it from the CLI " are you talking about the template or if the template is local or not local?
either way, getting the template is first step, the javascript needs to know if the template is remote or local.
if local it imports the template to hand off to fcl. 
if remote then the url is passed to fcl. Either way some place in the code needs to know if the template is a file reference or url.